app.js

```diff
@@ -319,7 +223,7 @@ function isValidRequest(requestData, poolExpiryTime, poolCreationTime) {
                 diffTime_Pool = Math.abs(pool_expiry_date - end_date)
                 diffDays_Pool = Math.ceil(diffTime_Pool / (1000 * 60 * 60 * 24))
 
-                if (diffDays >= 30 && diffDays_Pool >= 10 && start_date > pool_creation_date) {
+                if (diffDays >= 30 && diffDays_Pool >= 10) {
```

Comment:
Im wondering whether we have discussed and agreed to remove the start_date > pool_creation_date condition. I don’t think it’s wrong, I‘m just trying to understand the reason.